packages/next/src/client/components/app-router-instance.ts

```diff
@@ -44,6 +45,8 @@ export type AppRouterActionQueue = {
   last: ActionQueueNode | null
 }
 
+export type GlobalErrorState = [GlobalErrorComponent, React.ReactNode]
```

Comment:
What do we call these tuple members?
```suggestion
export type GlobalErrorState = [GlobalError: GlobalErrorComponent, styles: React.ReactNode]
```